frontend/app/controllers/user/index.js

```diff
@@ -3,11 +3,11 @@ import Ember from 'ember';
 export default Ember.ArrayController.extend({
   needs: "user",
   user: Ember.computed.alias('controllers.user'),
-  waifu_slug: Ember.computed.any('user.waifuSlug'),
-  hasWaifu: Ember.computed.any('user.waifu'),
-  hasLocation: Ember.computed.any('user.location'),
-  hasWebsite: Ember.computed.any('user.website'),
-  hasAboutText: Ember.computed.any('user.about'),
+  waifu_slug: Ember.computed.any('user.model.waifuSlug'),
```

Comment:
Underscores?  In _my_ JS?

It's more likely than you think!

`FREE PC CHECK`